coding-concepts.js

```diff
@@ -7,62 +7,87 @@
 
 // --------------------1) What will this log?
 
-const cohort = "Charlie 2023"
+// const cohort = "Charlie 2023"
 // console.log(cohort.split(""))
 
 // a) Your answer:
-// b) Verify and explain:
+  // ["C", "h", "a", "r", "l", "i", "e", " ", "2", "0", "2", "3"]
+// b) Verify and explain: return:
+  // ['C', 'h', 'a', 'r', 'l', 'i', 'e', ' ', '2', '0', '2', '3'],
+  // I was correct because a space does take an index so when using split method to trun string into an array, 
+  // every character including space will have quotation mark with it.
 
 // --------------------2) What will this log?
 
-const greeter = (name) => {
-  `Hello, ${name}!`
-}
+// const greeter = (name) => {
+//   `Hello, ${name}!`
+// // }
+
 // console.log(greeter("LEARN Student"))
 
 // a) Your answer:
-// b) Verify and explain:
+  // it will log nothing
+// b) Verify and explain: 
+  // return:undefined, 
+  // I was close because this function doesn't have return in it.
+  // // working code will be:
+  // const greeter = (name) => {
+  //   return `Hello, ${name}!`
+  //  }
 
 // --------------------3) What will this log?
 
-const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
+// const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
 // console.log(multipliedByTwo)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 
+  // undefine
+// b) Verify and explain:return:
+  // [ 8, 10, 12, 14, 16 ]
+  // I was wrong because I forgot in high order funtion => is short for returning value.
```

Comment:
It's not the arrow syntax making the return. When you write a function using arrow syntax without curly braces, the return is implied.